src/unpage/agent/utils.py

```diff
@@ -38,9 +39,13 @@ def get_agent_template(agent_name: str) -> str:
     return (Path(__file__).parent / "templates" / f"{agent_name}.yaml").read_text()
 
 
+class agentWithDescriptionOnly(BaseModel):
```

Comment:
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWhjNGcycGI5b24xdTdieHNnOWdwYWF2emRvN2N4c3A3NGNocmw0eSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xT8qB7GQDfnldWTFhS/giphy.gif)

```suggestion
class AgentWithDescriptionOnly(BaseModel):
```

Reply:
lol probably worth a specific formatting rule if we care about this sort of thing.